.github/workflows/pages.yml

```diff
@@ -17,6 +17,51 @@ jobs:
     uses: ./.github/workflows/reusable_kernel_tests.yaml
     secrets: inherit
 
+  perf-libs-master:
```

Comment:
`pages` CI is now responsible to run perf on master.
Moreover, it generates flamegraph svg files using https://github.com/brendangregg/FlameGraph, and uploads it to be later consumed by `deploy-pages` job.